### PR TITLE
Improving the custom cron documentation

### PR DESCRIPTION
**Pull request by eduard13**
## Purpose of this pull request

This pull request (PR) improves the custom cron documentation, by using a system config for the schedule.

## Affected DevDocs pages

-  https://devdocs.magento.com/guides/v2.3/config-guide/cron/custom-cron-tut.html

## Links to Magento source code

-  https://github.com/magento/magento2/blob/2.3/app/code/Magento/ProductAlert/etc/crontab.xml

### CODE REVIEW

**Comment by devops-devdocs** [comment]
An admin must run tests on this PR before it can be merged.

**Comment by devops-devdocs** [comment]
An admin must run tests on this PR before it can be merged.

File: src/guides/v2.3/config-guide/cron/custom-cron-tut.md [hunk 1]
```diff
@@ -174,6 +174,19 @@ Create `crontab.xml` as follows in the `/var/www/html/magento2/app/code/Magento/
...
 The preceding `crontab.xml` runs the `Magento/SampleMinimal/Cron/Test.php` class once per minute, resulting in a row being added to the `cron_schedule` table.
 
+In order to make the cron schedule configurable from admin panel, you may use the configuration path of your system configuration field.
+
+```xml
+<?xml version="1.0"?>
+<config xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="urn:magento:module:Magento_Cron:etc/crontab.xsd">
+    <group id="default">
+        <job name="custom_cronjob" instance="Magento\SampleMinimal\Cron\Test" method="execute">
+            <config_path>system/config/path</config_path>
```
Comment (rogyar) [inline src/guides/v2.3/config-guide/cron/custom-cron-tut.md]: Please, add a small hint regarding what `<config_path>` is. Like 

```
... where config_path is a system configuration path defined in `etc/adminhtml/system.xml` of a module. 
```

Thank you!

Comment (eduard13) [reply src/guides/v2.3/config-guide/cron/custom-cron-tut.md]: Thank you, updated.

Comment (rogyar) [reply src/guides/v2.3/config-guide/cron/custom-cron-tut.md]: Thank you!

File: src/guides/v2.3/config-guide/cron/custom-cron-tut.md [hunk 2]
```diff
@@ -174,6 +174,21 @@ Create `crontab.xml` as follows in the `/var/www/html/magento2/app/code/Magento/
 
 The preceding `crontab.xml` runs the `Magento/SampleMinimal/Cron/Test.php` class once per minute, resulting in a row being added to the `cron_schedule` table.
 
+In order to make the cron schedule configurable from admin panel, you may use the configuration path of your system configuration field.
```
Comment (dobooth) [inline src/guides/v2.3/config-guide/cron/custom-cron-tut.md]: "...configurable from the admin panel, use the...

Comment (eduard13) [reply src/guides/v2.3/config-guide/cron/custom-cron-tut.md]: Hi @dobooth, updated.
Thanks.

**Comment by dobooth** [comment]
running tests

**Pull request merged**